src/csi/v0_volume_manager.cpp

```diff
@@ -1075,15 +1075,15 @@ Future<Nothing> VolumeManagerProcess::__publishVolume(const string& volumeId)
 
   if (!volumeState.node_stage_secrets().empty()) {
     rpcResult = resolveSecrets(volumeState.node_stage_secrets())
-      .then([=](const Map<string, string>& secrets) {
```

Comment:
Nit: Should we use `process::defer` instead? I see `process::defer` is used widely in this file.

Reply:
Sure  - done.